src/methods/randomshuffle.jl

```diff
@@ -4,8 +4,21 @@ export RandomShuffle
 
 A random constrained surrogate, generated by shifting values around.
 
-This method destroys any linear
-correlation in the signal, but preserves its amplitude distribution.
+Random shuffle surrogates preserve the mean, variance and amplitude 
+distribution of the original signal. Properties not preserved are *any 
+temporal information*, such as the power spectrum[^Lancaster2018] 
+and hence linear correlations. 
+
+The null hypothesis this method can test for is whether the data 
+are uncorrelated noise, possibly measured via a nonlinear function.
+Specifically, random shuffle surrogate can test 
+the null hypothesis that the original signal is produced by independent and 
+identically distributed random variables[^Theiler1991, ^Lancaster2018]. 
+
+*Beware: random shuffle surrogates do not cover the case of correlated noise*[^Lancaster2018]. 
+
+[^Theiler1991]: J. Theiler, S. Eubank, A. Longtin, B. Galdrikian, J. Farmer, Testing for nonlinearity in time series: The method of surrogate data, Physica D 58 (1–4) (1992) 77–94.
```

Comment:
Here you should copy paste the exact way we use the reference in the other docstrings (so that they all point to a single reference at the end)

Reply:
Nice spot. The year was actually wrong for this reference. I've fixed this for all methods.